src/components/dashboard/FeedItems/ListEventItem.js

```diff
@@ -60,38 +55,63 @@ const ListEvent = ({ item: feed, theme, styles }: FeedEventProps) => {
 
 const getStylesFromProps = ({ theme }) => ({
   innerRow: {
-    padding: normalize(4),
+    alignItems: 'center',
     flexDirection: 'row',
     justifyContent: 'center',
-    alignItems: 'center',
+    padding: normalize(8),
     width: '100%',
   },
-  avatatBottom: {
+  avatarBottom: {
     alignSelf: 'flex-end',
   },
   mainSection: {
-    marginLeft: normalize(4),
+    marginLeft: normalize(8),
```

Comment:
shouldn't we use `theme.paddings` values here? There's a `padding` set for `innerRow` in this same object.

Reply:
No. Global styles have nothing to do here.

It may be that we could use a better name for this style / component, but having a local style for this is correct IMO.